internal/pkg/cli/pipeline_init.go

```diff
@@ -26,27 +28,35 @@ import (
 )
 
 const (
-	pipelineAddEnvPrompt            = "Would you like to add an environment to your pipeline?"
-	pipelineAddMoreEnvPrompt        = "Would you like to add another environment to your pipeline?"
-	pipelineAddEnvHelpPrompt        = "Adds an environment that corresponds to a deployment stage in your pipeline. Environments are added sequentially."
-	pipelineAddMoreEnvHelpPrompt    = "Adds another environment that corresponds to a deployment stage in your pipeline. Environments are added sequentially."
-	pipelineSelectEnvPrompt         = "Which environment would you like to add to your pipeline?"
-	pipelineEnterGitHubRepoPrompt   = "What is your application's GitHub repository?" // TODO allow just <user>/<repo>?
-	pipelineEnterGitHubBranchPrompt = "What is the branch name of your GitHub repository?"
+	pipelineAddEnvPrompt                 = "Would you like to add an environment to your pipeline?"
+	pipelineAddMoreEnvPrompt             = "Would you like to add another environment to your pipeline?"
+	pipelineAddEnvHelpPrompt             = "Adds an environment that corresponds to a deployment stage in your pipeline. Environments are added sequentially."
+	pipelineAddMoreEnvHelpPrompt         = "Adds another environment that corresponds to a deployment stage in your pipeline. Environments are added sequentially."
+	pipelineSelectEnvPrompt              = "Which environment would you like to add to your pipeline?"
+	pipelineSelectGitHubURLPrompt        = "Which GitHub repository would you like to use for your application?"
+	pipelineSelectGitHubURLHelpPrompt    = `The GitHub repository linked to your workspace.\nPushing to this repository will trigger your pipeline build stage.\nPlease enter full repository URL, e.g. "https://github.com/myCompany/myRepo", or the owner/rep, e.g. "myCompany/myRepo"`
```

Comment:
I think this will actually write "\n" in the string.
When you use the back quote character you can just press enter